setup.py

```diff
@@ -43,8 +43,11 @@ def version():
     "requests==2.22.0",
     "six==1.12.0",
     "ipaddress==1.0.22",
-    "k8s==0.13.0",
+    "k8s==0.21.0",
     "prometheus_client == 0.7.1",
+    "jinja2<3.1.0",
+    "itsdangerous==2.0.1",
+    "werkzeug==2.0.3",
```

Comment:
why these new dependencies?

Reply:
Was under the impression I had to pin these. I'll revert that.